SOURCES/aqc111u-1.3.3.0.patch

```diff
@@ -0,0 +1,4396 @@
+diff -uNr aqc111u-1.3.3.0/.gitlab-ci.yml aqc111u-1.3.3.0-patch/.gitlab-ci.yml
+--- aqc111u-1.3.3.0/.gitlab-ci.yml	2021-04-04 19:02:03.000000000 -0700
++++ aqc111u-1.3.3.0-patch/.gitlab-ci.yml	2021-04-18 15:50:07.000000000 -0700
+@@ -27,13 +27,13 @@
+   stage: checkpatch
+   <<: *checkpatch-full-only
+   script:
+-    - /usr/src/linux-headers-$(uname -r)/scripts/checkpatch.pl --no-tree -f aqc111.[c,h]
+-    
++    - /usr/src/linux-headers-$(uname -r)/scripts/checkpatch.pl --no-tree -f aqc111u.[c,h]
++
+ checkpatch-soft:
+   stage: checkpatch
+   <<: *checkpatch-soft-only
+   script:
+-    - /usr/src/linux-headers-$(uname -r)/scripts/checkpatch.pl --no-tree --ignore LINUX_VERSION_CODE -f aqc111.[c,h]
++    - /usr/src/linux-headers-$(uname -r)/scripts/checkpatch.pl --no-tree --ignore LINUX_VERSION_CODE -f aqc111u.[c,h]
+ 
+ build-project:
+   stage: build
+diff -uNr aqc111u-1.3.3.0/Makefile aqc111u-1.3.3.0-patch/Makefile
+--- aqc111u-1.3.3.0/Makefile	2021-04-04 19:02:03.000000000 -0700
++++ aqc111u-1.3.3.0-patch/Makefile	2021-04-18 15:49:10.000000000 -0700
+@@ -1,6 +1,6 @@
+ CURRENT	= $(shell uname -r)
+-TARGET	= aqc111
+-OBJS	= aqc111.o
++TARGET	= aqc111u
++OBJS	= aqc111u.o
+ MDIR	= drivers/net/usb
+ 
+ ifndef KDIR
+@@ -20,7 +20,7 @@
+ PWD = $(shell pwd)
+ DEST = /lib/modules/$(CURRENT)/kernel/$(MDIR)
+ 
+-obj-m      :=  $(TARGET).o  
++obj-m      :=  $(TARGET).o
+ 
+ default:
+ 	make -C $(BUILD_DIR) SUBDIRS=$(PWD) modules
+diff -uNr aqc111u-1.3.3.0/README.md aqc111u-1.3.3.0-patch/README.md
+--- aqc111u-1.3.3.0/README.md	2021-04-04 19:02:03.000000000 -0700
++++ aqc111u-1.3.3.0-patch/README.md	2021-04-18 15:49:01.000000000 -0700
+@@ -2,25 +2,25 @@
+ 
+ ## Build from sources:
+ * Unpack sources to temporary folder (e.g `tar -xf pacific.tar.gz`).
+-  It will create Linux/ directory 
++  It will create Linux/ directory
+ * Change directory (`cd Linux`)
+-* Perform `make` 
++* Perform `make`
+ 
+ ## Load driver:
+ * `modprobe usbnet`
+-* `insmod aqc111.ko`
++* `insmod aqc111u.ko`
+ 
+ ## Unload driver:
+-* `rmmod aqc111`
++* `rmmod aqc111u`
+ 
+ ## Install driver:
+ Note: starting from 5.x kernel, driver is built in. Be aware to corrupt original file.
+-* `cp aqc111.ko /lib/modules/$(uname -r)/kernel/drivers/net/usb/`
++* `cp aqc111u.ko /lib/modules/$(uname -r)/kernel/drivers/net/usb/`
+ * `depmod -a`
+ 
+ ## Uninstall driver:
+ Note: starting from 5.x kernel, driver is built in. Be aware to corrupt original file.
+-* `rm /lib/modules/$(uname -r)/kernel/drivers/net/usb/aqc111.ko`
++* `rm /lib/modules/$(uname -r)/kernel/drivers/net/usb/aqc111u.ko`
+ * `depmod -a`
+ 
+ ## Private flags reference:
+diff -uNr aqc111u-1.3.3.0/aqc111.c aqc111u-1.3.3.0-patch/aqc111.c
+--- aqc111u-1.3.3.0/aqc111.c	2021-04-04 19:02:03.000000000 -0700
++++ aqc111u-1.3.3.0-patch/aqc111.c	1969-12-31 16:00:00.000000000 -0800
```

Comment:
It did not occur to me during first review, but it will be tedious for you to update these file renaming patches over time (each time anything changes in the files), where a simple `mv` in the spec file would do the trick.

Reply:
Am not sure am following here, but I'd rather avoid logic in the spec file and have a patch to take care of as much as possible.

Reply:
As you wish, you will just have more work for each update :)